docs/changelog/83350.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 83350
```

Comment:
I wonder if we should remove this changelog entry?

Reply:
Done